Explorer/Assets/Scripts/Global/Dynamic/DynamicWorldContainer.cs

```diff
@@ -634,15 +634,21 @@ IMultiPool MultiPoolFactory() =>
             return (container, true);
         }
 
-        private static URN[]? ParseSelfForcedEmotes(IAppArgs appParams)
+        private static IReadOnlyList<URN>? ParseForcedEmotes(IAppArgs appParams, IReadOnlyList<string>? debugEmotes)
```

Comment:
I think its wrong to add the responsibility to this method. It should only do one thing, parse the urns and retrieve the list. If you want to add the debug emotes, then you should do it where its called. Something like:
```
List<URN> selfEmotes = new List<URN();

IReadOnlyList<URN>? emotesThroughParams = ParseForcedEmotes(bootstrapContainer.ApplicationParametersParser);

if (emotesThroughParams != null)
    selfEmotes.AddRange(emotesThroughParams);
    
if (bootstrapContainer.DebugSettings.EmotesToAddToUserProfile != null)
    selfEmotes.AddRange(bootstrapContainer.DebugSettings.EmotesToAddToUserProfile);
    
var selfProfile = new SelfProfile(container.ProfileRepository, identityCache, equippedWearables, wearableCatalog,
                emotesCache, equippedEmotes, forceRender, selfEmotes);
....
```

Reply:
your proposal sounds cleaner! nice 👍